.github/workflows/sync_docs.yml

```
@@ -1,4 +1,6 @@
 name: Sync the Docs
+run-name: >-
+  Sync the Docs: version ${{ inputs.version }}
```

Why not

```suggestion
run-name: Sync the Docs: version ${{ inputs.version }}
```

Is it because the `version` might contain a newline?

---

Nah, I just generally default to using `>-` / `|` for strings in yaml because they're less error prone than inline values (especially when quoting etc. is required, as in that case with the internal `:`).